### DISCUSSION TITLE
Can I map a single keyword to multiple actions?

### DISCUSSION THREAD

**Discussion by pvys** [JetBrains/ideavim#473] [Q&A] [answered]
I'd like to map 'gh' to (QuickJavaDoc) and (ShowErrorDescription), like showing error if cursor is on an error code, or showing doc if quick documentation can be loaded.
I tried set my .ideavimrc action text side by side and by newline, neither work.
`map gh <Action>(QuickJavaDoc)<Action>(ShowErrorDescription)` (side by side)
```
map gh <Action>(QuickJavaDoc)
map gh<Action>(ShowErrorDescription)  //(by newline)
```
Is this achievable?

**Comment by AlexPl292** [comment, marked as answer]
Hi, `ShowErrorDescription` shows a popup with an error message and java doc in it.

**Reply by kbrah** [reply]
I can't get the Java doc to show up with ShowErrorDescription. I only get the error message. Any idea what I'm doing wrong?

**Reply by citizenmatt** [reply]
Starting with IdeaVim 2.7.0, you can use the `ShowHoverInfo` which shows a combined popup of error message and quick doc description. Something like:

```
nmap gh <Action>(ShowHoverInfo)
```

**Comment by mikesmithgh** [comment]
FYI, in general, it appears you can map multiple actions by separating them with `\|`

This works for me:
```
:map <leader>lf <Action>(io.gitlab.arturbosch.detekt.idea.action.AutoCorrect) \| <Action>(ReformatCode) 
```
